### PR TITLE
Feature/calculate token amounts from lp tokens

### PR DESCRIPTION
**Pull request by evandrosaturnino**
- calculateTokenAmountsFromLPTokens function
- calculateShareOfPoolProvided - Already provided share
- getSymbol on ERC20

### CODE REVIEW

File: src/exchange/Exchange.mjs [hunk 1]
```diff
@@ -155,6 +174,17 @@ export default class Exchange extends Base {
     return newTokens.div(totalSupplyOfLiquidityTokens.plus(newTokens));
   }
 
+  async calculateShareOfPoolProvided(lpAmount, lpTokenDecimals) {
```
Comment (0xean) [inline src/exchange/Exchange.mjs:177]: why is the lpTokenDecimals needed here? this should already be in base units?

**Review by 0xean — COMMENTED** [review]
one small question otherwise 👍

File: src/exchange/Exchange.mjs (hunk 1 above)
Comment (evandrosaturnino) [reply src/exchange/Exchange.mjs]: resolved

**Comment by evandrosaturnino** [comment]
> one small question otherwise +1
Addressed

**Pull request merged**